packages/fuels-test-helpers/src/lib.rs

```diff
@@ -185,27 +184,25 @@ pub async fn setup_test_client(
     node_config: Option<Config>,
     consensus_parameters_config: Option<ConsensusParameters>,
 ) -> (FuelClient, SocketAddr) {
-    let (srv_address, manual_blocks_enabled) = match node_config {
-        Some(config) if config.addr.port() != 0 && is_free(config.addr.port()) => {
-            (config.addr, config.manual_blocks_enabled)
-        }
+    let config = match node_config {
```

Comment:
This [comment](https://github.com/FuelLabs/fuels-rs/pull/488#discussion_r930118897) might apply here.